Update to build in linux

clebail opened the pull request.

clebail committed:
Update to build in linux

clebail committed:
Color file

ikoruk commented:
Thanks for the PR! However, `#include <GL/glu.h>` breaks mac compatibility. Can you please add a check for mac/linux and then we'll gladly accept the change. Thanks again!

Commit:
Check for mac/linux for glu.h
Add -std=c++11 to avoid the error "error: 'shared_ptr' in namespace 'std' does not name a template-type qmake"
